pkg/cmd/pipelinerun/logs_test.go

```diff
@@ -2095,6 +2120,7 @@ func logOpts(name string, ns string, cs pipelinetest.Clients, dc dynamic.Interfa
 }
 
 func fetchLogs(lo *options.LogOptions) (string, error) {
+	fmt.Println("Limit is -", lo.Limit)
```

Comment:
This looks like it was from debugging. Please remove it.

Reply:
Removed it, Sorry about that.